smrf/data/loadGrid.py

```diff
@@ -63,18 +59,16 @@ def __init__(self, dataConfig, topo, start_date, end_date,
         # the model domain are used
         self.x = topo.x
         self.y = topo.y
-        self.lat = topo.topoConfig['basin_lat']
-        self.lon = topo.topoConfig['basin_lon']
 
-        # get the zone number and the bounding box
-        u = utm.from_latlon(topo.topoConfig['basin_lat'],
-                            topo.topoConfig['basin_lon'],
-                            self.force_zone_number)
-        self.zone_number = u[2]
-        self.zone_letter = u[3]
+        self.lat = topo.basin_lat
+        self.lon = topo.basin_long
+        self.zone_number = topo.zone_number
+        self.northern = topo.northern_hemisphere
```

Comment:
Can we make all of these variables on `self` more verbose? Or are they even needed? If we put `topo` on `self` can we just call `self.topo.basin_lat`? Then it's more verbose and we're not copying variables.

Reply:
They are needed. Would you rather just assign `self.topo = topo` and then use it that way in all the place its used? e.g. 

```
 ur = np.array(utm.to_latlon(np.max(self.x), np.max(self.y), self.zone_number, northern=self.northern))
```
would become 

```
 ur = np.array(utm.to_latlon(np.max(self.topo.x), np.max(self.topo.y), self.topo..zone_number, northern=self.topo.northern_hemisphere))
```

When I first wrote it I was aiming to reduce the number of lines modified. So this will have a slightly farther reach in the file.

Reply:
This is the way we use it elsewhere as well, it makes it cleaner and more consistent

Reply:
Sounds good.

Reply:
Ultimately all the topo related info could be moved into it's own class and have all the related calculations done there. Makes easier testing as well.
This is long term wish list thinking ....